ngraph/frontend/onnx_import/src/op/org.openvinotoolkit/prior_box.cpp

```diff
@@ -83,6 +83,35 @@ namespace ngraph
                         axes)};
                 }
 
+                OutputVector prior_box_clustered(const Node& node)
+                {
+                    auto inputs = node.get_ng_inputs();
+                    NGRAPH_CHECK(inputs.size() == 2, "Invalid number of inputs");
+
+                    auto output_shape = std::make_shared<default_opset::ShapeOf>(inputs[0]);
+                    auto image_shape = std::make_shared<default_opset::ShapeOf>(inputs[1]);
+                    auto output_shape_slice = detail::make_slice(output_shape, 2, 4);
+                    auto image_shape_slice = detail::make_slice(image_shape, 2, 4);
+
+                    ngraph::op::PriorBoxClusteredAttrs attrs{};
+                    attrs.widths = node.get_attribute_value<std::vector<float>>("width", {1.0});
+                    attrs.heights = node.get_attribute_value<std::vector<float>>("height", {1.0});
+                    attrs.clip = static_cast<bool>(node.get_attribute_value<int64_t>("clip", 0));
```

Comment:
In the [PriorBoxClustered]( https://docs.openvinotoolkit.org/latest/openvino_docs_ops_detection_PriorBoxClustered_1.html ) default value for the clip attr is `True`
```suggestion
                    attrs.clip = static_cast<bool>(node.get_attribute_value<int64_t>("clip", 1));
```

Reply:
I have followed Model Optimizer implementation:
openvino/model-optimizer/extensions/front/onnx/priorbox_clustered_ext.py:37
`'clip': onnx_attr(node, 'clip', 'i', default=0),`

I think we should follow MO implementations of custom ONNX operators rather than documentation of the operator we are using.

Reply:
I think that the spec behavior is what user will expect, if there is mismatch between code and the spec we should try to resolve discrepancies.

Reply:
If there is an official documentation for such op we should follow it IMHO

Reply:
@mateusztabaka, @postrational  - what do you think?

Reply:
This is the documentation for OpenVINO op, not ONNX operator. What ONNX models expect are the default values from model-optimizer/extensions/front/onnx/priorbox_clustered_ext.py

Reply:
This is not an ONNX op, but a custom op used by OpenVINO. We should follow OpenVINO's previous implementation for this.

Reply:
:shipit: